app/models/signing_certificate_event.rb

```diff
@@ -4,6 +4,7 @@ class SigningCertificateEvent < AggregatedEvent
   belongs_to_aggregate :certificate
   validates_presence_of :certificate
   validate :certificate_is_signing?
+  after_save :trigger_publish_event
```

Comment:
I think we should consider introducing a class_method for this so keep the code DRY.